File: 1672-richest-customer-wealth/1672-richest-customer-wealth.py
class Solution:
    @staticmethod
    def totalBalanceCheck(x):
        sum_list = 0
        for i in x:
            sum_list += i
        return sum_list

    def maximumWealth(self, accounts):
        res = []
        for i in accounts:
            res.append(self.totalBalanceCheck(i))
        res.sort()
        return res[-1]

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.go
func maximumWealth(accounts [][]int) int {
    max_res := -9999999999
    res := 0
    for i:=0;i<len(accounts);i++{
        res = 0
        for j:=0;j<len(accounts[i]);j++{
            res += accounts[i][j]
        }
        if res > max_res{
            max_res = res
        }
    }
    return max_res
}